pkg/webhook/admission/webhook.go

```diff
@@ -92,6 +92,10 @@ func (r *Response) Complete(req Request) error {
 
 // Handler can handle an AdmissionRequest.
 type Handler interface {
+	// Handle yields a response to an AdmissionRequest.
+	//
+	// The supplied context is extracted from the received http.Request, allowing wrapping
+	// http.Handlers to inject values into and control cancelation of downstream request processing.
```

Comment:
In proxy server terms, the inner handlers are analogous to "upstream" servers. Given that, if my use of "downstream" is confusing, please let me know and I can try again.

Reply:
eh, I think it's clear enough if you're vaguely familiar with context.  We'll see if anyone complains :-)